### DISCUSSION TITLE
Ability to hide some accounts from the client's account list

### DISCUSSION THREAD

**Discussion by JokerGermany** [element-hq/element-meta#2006] [Ideas]
#### What would you like to do?
privacy
e.g:
The use case is that in some countries this is happens that police stops people and asks / force them to unlock the phone and show the list of subscriptions. They are looking mostly for opposition rooms / chats. It may result in 7 years of imprisonment in Belarus just for subscription to an opposition chat or channel, but this is not the only country with this kind of disrespect to privacy and presumption of innocence.

#### Why would you like to do it?
Hiding an account with dangerous content will safe someone's freedom. For example, the biggest opposition channel in Belarus has more than million subscribers, it's more than 10% of population of the country.
#### How would you like to achieve it?
Example of similar functionality can be observed in SimpleX chat client. There you need to input your secret password into Searching field, after it the hidden account appears in the list, otherwise there is no traces of having anything hidden.

### Additional context

It's "stolen" from https://github.com/vector-im/element-x-android/issues/190#issuecomment-1538175788 
I don't need it, but i think after multiple accounts is implemented this could be a nice feature and I don't want that this get lost in vector-im/element-x-android#190 

### Are you willing to provide a PR?

No

**Comment by kittykat** [comment]
Transferring to discussions as this is cross-platform

**Comment by shadow-identity** [comment]
Thanks for moving it form the comment.
